server.py

```diff
@@ -57,9 +62,16 @@ def post_collect():
     workers.download_job(input_data['url'], source_id, next_service)
     APP.logger.info('Job started.')
 
+    prometheus_metrics.METRICS['jobs_initiated'].inc()
     return jsonify(status="OK", message="Job initiated")
 
 
+@APP.route("/metrics", methods=['GET'])
```

Comment:
You're adding an endpoint here which is not described in the `swagger.yml`. Can you please update that description as well?

Reply:
oops...yes, thanks. 
Just added.

Also updated the PR description.